src/content/docs/setup/seo/indexing.mdx

```diff
@@ -35,24 +37,60 @@ The URL format for folder mapped PDPs looks like this:
 This is to include the SEO relevant `urlKey` parameter and the `SKU` which is needed to perform Catalog Service queries. Feel free to customize the format as needed. You might need to do advanced customization on the CDN.
 
 :::note[Uppercase characters in URLs]
-Be aware that Edge Delivery Services does not handle URLs with uppercase letters well. This can cause issues if any of your product SKUs contain uppercase characters.
+Be aware that Edge **Delivery** Services does not handle URLs with uppercase letters well. This can cause issues if any of your product SKUs contain uppercase characters.
```

Comment:
Is double asterisk bold? Why did you bold `Delivery`?

Reply:
It is, but it was unintentional. I must've inadvertently highlighted and pressed a hotkey. I'll revert.